Rubberduck.Inspections/QuickFixes/AddIdentifierToWhiteListQuickFix.cs

```diff
@@ -27,13 +27,10 @@ public override void Fix(IInspectionResult result)
             _settings.Save(inspectionSettings);
         }
 
-        public override string Description(IInspectionResult result)
-        {
-            return InspectionsUI.WhiteListIdentifierQuickFix;
-        }
+        public override string Description(IInspectionResult result) => InspectionsUI.WhiteListIdentifierQuickFix;
 
-        public override bool CanFixInProcedure { get; } = false;
-        public override bool CanFixInModule { get; } = false;
-        public override bool CanFixInProject { get; } = false;
+        public override bool CanFixInProcedure => false;
```

Comment:
I sure hope you realize the difference between these... It's not a problem here, but you'd better know it because it will result in different values in certain cases.

Reply:
Also, I prefer the first here.